README.md

```diff
@@ -1,6 +1,8 @@
 # Rain World Hunter Tracker
 Allows progress tracking for the Hunter in Rain World.
 
+This tracker only works for the Steam edition of the game. If you have the PS4/Switch edition, it will not work.
```

Comment:
Mabye next version there could be ps4/switch compatibility

Reply:
Thanks for all the support with the tracker. However this application of the tracker is not compatible with consoles and it isn't within scope to make them.
This application only reads the save files and parses out their data into a visual format.

Reply:
Can you merge this pull request?

Reply:
Also arent the console version savefiles the same format?

Reply:
I don't know I've never seen one.
It would be a little disingenuous to say that this application was made for PS4/Switch, as the repo is the backbone for the website and a Windows-only version of the tracker app made in Electron.

The website is here: https://jaclynonacloud.github.io/rainworld-tracker/
The application is here: https://github.com/jaclynonacloud/rainworld-tracker/releases

I'll accept your PR. : )